wallet/how-to/use-non-evm-networks/starknet/manage-starknet-accounts.md

```diff
@@ -0,0 +1,419 @@
+---
+description: Manage Starknet accounts in MetaMask.
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Manage Starknet accounts
```

Comment:
Since the examples on this page display information in the dapp, it would be nice to include screenshots. Can be raised for a future PR.